osu.Game/Graphics/UserInterface/ShearedToggleButton.cs

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Input.Events;
+using osu.Framework.Localisation;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class ShearedToggleButton : OsuClickableContainer
+    {
+        public BindableBool Active { get; } = new BindableBool();
+
+        public LocalisableString Text
+        {
+            get => text.Text;
+            set => text.Text = value;
+        }
+
+        private readonly Box background;
+        private readonly OsuSpriteText text;
+
+        private Sample? sampleOff;
+        private Sample? sampleOn;
+
+        [Resolved]
+        private OverlayColourProvider colourProvider { get; set; } = null!;
+
+        public ShearedToggleButton(float shear)
```

Comment:
Are there going to be multiple `shear` amounts in the design? If not, I'd hardcode this.

Reply:
I don't foresee any, was a bit of future-proofing at best. I'll remove for now.